Standards/scs-0402-v1-status-page-openapi-spec-decision.md

```diff
@@ -137,6 +135,12 @@ A severity with the value of 100 MUST always be supplied. This is the highest se
 
 Components list their impacts, which they are affected by, as read only. Only an incident creates an impact on a component. Components MUST only list their currently active impacts.
 
+An optional `now` parameter can be supplied, to set a reference time to show all incidents, active at that time, even when they are inactive currently.
```

Comment:
Wrong comma placement.

Reply:
```suggestion
An optional `now` parameter can be supplied, to set a reference time to show all incidents, active at that time, even when they are inactive currently.
```

Reply:
Have the docs ever defined what an incident being "active" means? If not, we should add this definition, just to make it abundantly clear.

Reply:
No, thats true. 

An active incident is just an incident without a `ended_at` set.

Reply:
Mentioned the incidents and their end condition.